resources/r/model.R

```diff
@@ -62,10 +62,11 @@ manipulate_regions_for_shapefile <- function(region_prevalence) {
   
 #Add rank column/variable to dataset
 
-create_rank_variable_by_region <- function(thirteen_level_NHS_regional_prevalence){
+rank_prevalence_by_region <- function(thirteen_level_NHS_regional_prevalence){
   thirteen_level_NHS_regional_prevalence$rank <- NA
   thirteen_level_NHS_regional_prevalence$rank[order(-thirteen_level_NHS_regional_prevalence$prevalence)] <- 1:nrow(thirteen_level_NHS_regional_prevalence)
-  return(thirteen_level_NHS_regional_prevalence)
+  
+  return(regional_prevalence_with_ranks)
```

Comment:
This doesn't seem to be working now you've changed it from 'thirteen_level_nhs_regional_prevalence', 'regional_prevalence_with_ranks' isn't used anywhere in the function so it can't return it?